checkstyle.xml

```diff
@@ -23,6 +23,8 @@
   <property name="severity" value="error"/>
 
   <property name="fileExtensions" value="java, properties, xml"/>
+  
+  <property name="cacheFile" value="${build.dir}/checkstyle_cachefile"/>
```

Comment:
oh, I'm sorry :-) I have just seen a lot of comments related to this

Reply:
I think you're right. I missed that folder. They belong in there imo and it will still be under build. So all good. I have committed your suggestion. Thx.